pkg/taskrun/taskrun.go

```diff
@@ -68,17 +90,20 @@ func SortTasksBySpecOrder(pipelineTasks []v1beta1.PipelineTask, pipelinesTaskRun
 	for name, t := range pipelinesTaskRuns {
 		trNames[t.PipelineTaskName] = name
 	}
+	trs := Runs{}
 
-	trs := []Run{}
 	for _, ts := range pipelineTasks {
 		if n, ok := trNames[ts.Name]; ok {
+			trStatusFields := pipelinesTaskRuns[n].Status.TaskRunStatusFields
```

Comment:
There are two test failing 
[TestPipelinerunLog_completed_taskrun_only](https://github.com/tektoncd/cli/blob/c2f113546808911c915cdfd34789233f34a61ae6/pkg/cmd/pipelinerun/logs_test.go#L1328)
and 
[TestPipelinerunLog_completed_taskrun_only_v1beta1](https://github.com/tektoncd/cli/blob/c2f113546808911c915cdfd34789233f34a61ae6/pkg/cmd/pipelinerun/logs_test.go#L1328)
`pipelinesTaskRuns[n].Status.TaskRunStatusFields` field is required in pipelinerun test struct 
I tried adding it for [v1beta1 test ](https://gist.github.com/pradeepitm12/999f9f853e1cc9c1f65d6c7befd69b05) and it works. 
we can try adding the fields to [TestPipelinerunLog_completed_taskrun_only](https://github.com/tektoncd/cli/blob/c2f113546808911c915cdfd34789233f34a61ae6/pkg/cmd/pipelinerun/logs_test.go#L1328) also but we can remove it as this is same as the other one only diff of version v1alpha1 v1beta1

Reply:
Thanks 👍 
resolved 🧑‍💻